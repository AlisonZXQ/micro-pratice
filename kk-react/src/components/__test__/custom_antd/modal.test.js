import React from 'react';
import { shallow } from 'enzyme';
import mountTest from '@tests/shared/mountTest';
import Modal from '../../CustomAntd/modal';

describe('modal', () => {
  mountTest(Modal);

  it('should snapshot correctly', () => {
    const shallowWrapper = shallow(<Modal />);
    expect(shallowWrapper).toMatchSnapshot();
  });
})

