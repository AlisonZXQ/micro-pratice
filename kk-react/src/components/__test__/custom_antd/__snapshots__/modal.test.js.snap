// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal should snapshot correctly 1`] = `
<Fragment>
  <Modal
    confirmLoading={false}
    maskClosable={false}
    maskTransitionName="fade"
    okType="primary"
    transitionName="zoom"
    visible={false}
    width={520}
  />
</Fragment>
`;
